modules/glslang/register_types.cpp

```diff
@@ -52,10 +52,20 @@ static Vector<uint8_t> _compile_shader_glsl(RenderingDevice::ShaderStage p_stage
 
 	int ClientInputSemanticsVersion = 100; // maps to, say, #define VULKAN 100
 
-	glslang::EShTargetClientVersion VulkanClientVersion = glslang::EShTargetVulkan_1_0;
-	glslang::EShTargetLanguageVersion TargetVersion = glslang::EShTargetSpv_1_3;
+	glslang::EShTargetClientVersion VulkanClientVersion = glslang::EShTargetVulkan_1_2;
+	glslang::EShTargetLanguageVersion TargetVersion = glslang::EShTargetSpv_1_5;
 	glslang::TShader::ForbidIncluder includer;
 
+	if (p_vulkan_major == 1 && p_vulkan_minor == 0) {
+		VulkanClientVersion = glslang::EShTargetVulkan_1_0;
+		TargetVersion = glslang::EShTargetSpv_1_0;
+	} else if (p_vulkan_major == 1 && p_vulkan_minor == 1) {
+		VulkanClientVersion = glslang::EShTargetVulkan_1_1;
+		TargetVersion = glslang::EShTargetSpv_1_3;
+	} else {
```

Comment:
According to Khronos Vulkan 1.1 can use SpirV 1.3, but this crashes on my phone. 
If I change this to SpirV 1.2 it runs.

Reply:
@clayjohn So far the documentation on Khronos seems to state that any Vulkan 1.1 capable device should support up to and including SpirV 1.3, so far we have not found any mechanism that tells us which version of SpirV is supported other then making this assumption.